### ISSUE TITLE
Broken in all browsers

### ISSUE THREAD

**Issue by randyaskin** [PredixDev/px-code-editor#1]
The demo code editor is completely broken, the modal opens but there's nothing inside it.

**Comment by randyaskin** [comment]
Once you click inside the code editor, this triggers a re-layout and everything appears fine. I was able to get the same behavior by adding this at the bottom of the `_initializeEditor` method, but I'm not sure if it's the best workaround. @davidrleonard any thoughts?

```
setTimeout(function() {
  this.editor.execCommand('goDocEnd');
}.bind(this));
```

**Comment by davidrleonard** [comment]
Fixes in 5c11a0458383d193160987b8e43e3e815797d196, released in v4.0.3.
